gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/datafetcher/EntityInteractionsFetcher.java

```diff
@@ -358,7 +358,7 @@ private void addInteractions(
                 QueryRequestUtil.createBetweenTimesFilter(
                     metadataProvider
                         .getAttributeMetadata(
-                            requestContext, AttributeScope.INTERACTION, TIMESTAMP_COLUMN_KEY)
+                            requestContext, AttributeScope.INTERACTION.name(), TIMESTAMP_COLUMN_KEY)
```

Comment:
Why are we still using INTERACTION enum?

Reply:
Yes, because this doesn't come from the user request. This is implicit in the entities request when user asks for incoming and outgoing edges right?

Reply:
So it seems to me that we are moving away completely from AttributeScope enum and this is hardcoded over here. Why can't we use a string const?